yogstation/code/modules/jungleland/jungle_alpha_mobs.dm

```diff
@@ -352,7 +352,7 @@
 	attacktext = "stings"
 	attack_sound = 'sound/voice/moth/scream_moth.ogg'
 	deathmessage = "rolls over, falling to the ground."
-	gold_core_spawnable = HOSTILE_SPAWN
+	gold_core_spawnable = NO_SPAWN
```

Comment:
just delete the line entirely, the parent typepath has it as FALSE already

Reply:
ok